LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Use the dart standard license. Copy it from https://github.com/dart-lang/shelf/blob/master/LICENSE w/ updated year

Reply:
done.